homeassistant/components/cookidoo/icons.json

```
@@ -0,0 +1,12 @@
+{
+  "entity": {
+    "todo": {
+      "ingredient_list": {
+        "default": "mdi:cart-plus"
```

```suggestion
        "default": "mdi:food-variant"
```
just a suggestion, so they are distinguishable visually

---

As it is rather generic as `additional items`, I would just leave it the same and let the user customize it if needed. In the app, both lists are merged.